.gitignore

```diff
@@ -13,4 +13,5 @@
 .externalNativeBuild
 .cxx
 local.properties
-/rates-cache/target
\ No newline at end of file
+/rates-cache/target
```

Comment:
`rates-cache` looks like coming from older version, now at this moment we don't have such a sub-folder in Git, do we?